app.py

```diff
@@ -25,13 +27,19 @@
     trigger=trigger,
     name="fetch_courses",
 )
+scheduler.add_job(
+    fine_tune_model,
+    trigger=trigger,
+    name="fine_tune_model",
+)
```

Comment:
```suggestion

scheduler.add_job(
    fine_tune_model,
    trigger=trigger,
    name="fine_tune_model",
)

```

Reply:
What is the difference with the code that is already there? Isn't it the same?

Reply:
Just two lines to pad it